### PR TITLE
fix(event): #MA-947 fix lateness without reason no show in scroll

### PR DESCRIPTION
**Pull request by BastienBenard**
## Describe your changes
When you scroll in the list of events, you call the API to get the following events. The problem is that the value of the filter was not used for delays without a reason.

## Checklist tests
Have a lateness without cause as well as an absence without reason and more than 20 events more recent than that of the lateness and the absence

## Issue ticket number and link
https://entsupport.gdapublic.fr/browse/MA-947

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: presences/src/main/resources/public/ts/controllers/event-list.ts [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+    EventListCalendarFilter,
+    EventResponse,
+    Events,
+    EventType,
+    IEvent,
+    IEventFormBody,
+    IStructureSlot,
+    Student,
+    TimeSlotHourPeriod
+} from '../models';
```
Comment (BastienBenard) [inline presences/src/main/resources/public/ts/controllers/event-list.ts:15]: indentation

File: presences/src/main/resources/public/ts/controllers/event-list.ts [hunk 2]
```diff
@@ -1,14 +1,27 @@
...
 import {DateUtils, PreferencesUtils, PresencesPreferenceUtils} from '@common/utils';
 import {Group, GroupService} from '@common/services/GroupService';
 import {actionService, EventRequest, EventService, ReasonService, ViescolaireService} from '../services';
 import {EventsFilter, EventsFormFilter, EventsUtils} from '../utilities';
 import {Reason} from '@presences/models/Reason';
 import {INFINITE_SCROLL_EVENTER} from '@common/core/enum/infinite-scroll-eventer';
 import {
-    ABSENCE_FORM_EVENTS, EVENTS_DATE,
+    ABSENCE_FORM_EVENTS,
+    EVENTS_DATE,
```
Comment (BastienBenard) [inline presences/src/main/resources/public/ts/controllers/event-list.ts:24]: indentation

File: presences/src/main/resources/public/ts/controllers/event-list.ts [hunk 3]
```diff
@@ -362,25 +377,7 @@ export const eventListController = ng.controller('EventListController', ['$scope
...
-                // If neither absences nor lateness are selected, the list of reasons is empty
-                listReasonIds: (vm.filter.regularized || vm.filter.notRegularized || vm.filter.late) ? vm.filter.reasonIds.toString() : '',
-                userId: vm.events.userId,
-                classes: vm.events.classes,
-                page: vm.interactedEvent.page
-            };
-            filter.regularized = (!(<any> vm.eventType).includes(1)) ? null : vm.filter.regularized;
-            filter.followed = (!(<any> vm.eventType).includes(1)) ? null : vm.filter.followed;
-            filter.notFollowed = (!(<any> vm.eventType).includes(1)) ? null : vm.filter.notFollowed;
-            let events = await eventService.get(filter);
```
Comment (BastienBenard) [inline presences/src/main/resources/public/ts/controllers/event-list.ts:383]: Code refacto dans createEventRequest

File: presences/src/main/resources/public/ts/controllers/event-list.ts [hunk 4]
```diff
@@ -816,25 +813,9 @@ export const eventListController = ng.controller('EventListController', ['$scope
...
-                eventType: vm.events.eventType,
-                // If neither absences nor lateness are selected, the list of reasons is empty
-                listReasonIds: (vm.filter.regularized || vm.filter.notRegularized || vm.filter.late) ? vm.filter.reasonIds.toString() : "",
-                followed: vm.events.followed,
-                notFollowed: vm.events.notFollowed,
-                userId: vm.events.userId,
-                classes: vm.events.classes,
-                page: vm.filter.page
-            };
-            filter.regularized = (!(<any> vm.eventType).includes(1)) ? null : vm.filter.regularized;
```
Comment (BastienBenard) [inline presences/src/main/resources/public/ts/controllers/event-list.ts:835]: Code refacto dans createEventRequest

**Review by flmartineau — APPROVED** [review]

**Review by alicedraillard — APPROVED** [review]

**Review by qpic-cgi — APPROVED** [review]

**Review by JinDouang — APPROVED** [review]

**Pull request merged**